python/py-pyqt5-webengine/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           python 1.0
 name                py-pyqt5-webengine
 python.rootname     PyQtWebEngine
 version             5.15.6
-revision            0
+revision            1
```

Comment:
this shouldn't require a "revision" increase as it only fixes a build issue for the Python 3.11 subport. Other than that I approve this intermediate fix until we have figured out to change the python PG.

Reply:
The build didn't fail earlier, it just generated bogus results. A rebuild seems in order:

```
$ curl -s https://packages.macports.org/py311-pyqt5-webengine/py311-pyqt5-webengine-5.15.6_0.darwin_22.arm64.tbz2 | tar jt
./
./+PORTFILE
./+CONTENTS
./+DESC
./+STATE
./opt/
./+COMMENT
./opt/local/
./opt/local/share/
./opt/local/share/doc/
./opt/local/share/doc/py311-pyqt5-webengine/
./opt/local/share/doc/py311-pyqt5-webengine/README
./opt/local/share/doc/py311-pyqt5-webengine/LICENSE
./opt/local/share/doc/py311-pyqt5-webengine/NEWS
./opt/local/share/doc/py311-pyqt5-webengine/ChangeLog
```

Reply:
good point - from memory I thought it failed but didn't actually verify that. Good to go then - thanks!